### DISCUSSION TITLE
Ignoring FBT003 for a specific function

### DISCUSSION THREAD

**Discussion by Apakottur** [astral-sh/ruff#6302] [General]
Consider the following code:
```python
from sqlalchemy import select

from sqlalchemy.orm import DeclarativeBase

class Base(DeclarativeBase):
    pass

class Table(Base):
    col: bool

select(Table).where(Table.col.is_(False))
```

Running Ruff with
```shell
ruff check --select FBT003 test.py
```

Correctly yields:
```
test.py:14:35: FBT003 Boolean positional value in function call
Found 1 error.
```

In almost all cases I do want Ruff to error on positional boolean values, but for this specific function I don't, because it only expects one argument, which is a boolean and it's very clear what's intended.

Is there a way to add the function `is_` to some kind of an allow-list for this check, so that I don't have to add a `noqa` every time we use it?

**Comment by zanieb** [comment]
Hey @Apakottur

We can include this in the exclusions for the rule https://github.com/astral-sh/ruff/pull/6307

**Reply by 93578237** [reply]
Could it be done also for Django ORM? (like [Value](https://docs.djangoproject.com/en/4.2/ref/models/expressions/#value-expressions))
